### PR TITLE
Fix warning caused by to_int32

### PR DESCRIPTION
**Pull request by yongtang**
While running tf.keras I noticed the following warning:
```
WARNING:tensorflow:From /usr/local/lib/python2.7/dist-
  packages/tensorflow/python/ops/math_ops.py:3064: 
  to_int32 (from tensorflow.python.ops.math_ops) is 
  deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
```

This fix fixes the warning caused by deprecated to_int32.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @yifeif: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Review by yifeif — APPROVED** [review]

**Comment by GbengaOdesanmi** [comment]
Please can someone help me on how to solve this? Thanks in advance

WARNING:tensorflow:From /home/mypath/envs/tensorflow/lib/python3.7/site-packages/tensorflow/python/ops/math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
(tensorflow)

**Comment by yongtang** [comment]
@GbengaOdesanmi The issue has been resolved and will be part of the next release. In the mean time, it is just a warning so it will not actually impact your application run.
